CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required (VERSION 3.6)
 project (vdf-Parser)
 
+add_library(ValveFileVDF INTERFACE)
+target_include_directories(ValveFileVDF INTERFACE "include")
```

Comment:
Can you change the target name to include a namespace and alias it?

I think exporting ValveFileVDF::ValveFileVDF is the standard and changing it at a later point would not be nice to people who use FetchContent, as they have to change the linking target again.

Perhaps another target like the python target might be added at a later point.

I would change target_include_directories change to HEADERS_SETS, but this is not important for this PR and can be done at a later point.

Reply:
I don't think that the target name can include a namespace by default, as when I am doing

```cmake
add_library(ValveFileVDF::ValveFileVDF INTERFACE)
```

CMake throws an error

```
  The target name "ValveFileVDF::ValveFileVDF" is reserved or not valid for
  certain CMake features, such as generator expressions, and may result in
  undefined behavior.
```

Although we can make the name `ValveFileVDF` and alias `ValveFileVDF::ValveFileVDF` to it, how does that sound?

Reply:
Thanks, thats sounds fine for me.